ethdb/database.go

```diff
@@ -78,6 +78,7 @@ func NewLDBDatabase(file string, cache int, handles int) (*LDBDatabase, error) {
 		BlockCacheCapacity:     cache / 2 * opt.MiB,
 		WriteBuffer:            cache / 4 * opt.MiB, // Two of these are used internally
 		Filter:                 filter.NewBloomFilter(10),
+		NoSync:                 true,
```

Comment:
From what I can tell, this field is always checked in combination with `WriteOptions.Sync`, so as longs as the latter is `false`, there should be no fsync. The only exception I could find was in `session.flushManifest`, where only the `GetNoSync` was used to determine whether to fsync.

Reply:
Good find! This is called on every table compaction, so it makes sense that if our tables are bigger, `fsync` would be called less often.

Reply:
Hmm.. In my tests I always have compaction disabled, because it looked like compaction caused more overhead than just leaving the db alone.